versioncontrol/peer.go

```diff
@@ -178,8 +218,64 @@ func (peer *Peer) Close() (err error) {
 // Addr returns the public address.
 func (peer *Peer) Addr() string { return peer.Server.Listener.Addr().String() }
 
-func configToProcess(binary Binary) version.Process {
-	return version.Process{
+// ValidateRollouts validates the rollout field of each field in the Versions struct.
+func (versions Versions) ValidateRollouts(log *zap.Logger) error {
+	value := reflect.ValueOf(versions)
```

Comment:
What's the purpose of using reflection?

Reply:
The original versioncontrol code used reflection to iterate over the processes like this. My assumption was that this was to make it more flexible if we change things going forward.

Reply:
OK, if it was that way, I agree to leave it as it was, I would have done the same.

From my point of view, there is no need for that flexibility in this code and using reflection makes harder to understand the code.